### ISSUE TITLE
Request for wandb report to compare loss curves

### ISSUE THREAD

**Issue by nuts-kun** [SWivid/F5-TTS#9]
Hi, Thank you for sharing this great work!
We have started to train this model on a multi-lingual dataset, including Japanese, and would be happy to compare it with the loss curve of the model trained for the paper to see how the learning progresses.
Would it be possible for you to share the wandb report?
Best regards.

**Comment by nuts-kun** [comment]
Thank you very much for the quick response!
Comparing the shared loss curve with the one from my model, it seems that the training is progressing smoothly :)
<img width="350" alt="image" src="https://github.com/user-attachments/assets/7327df61-2085-43d3-99b6-a3c5ce4b8d58">
Thank you again!

**Comment by SWivid** [comment]
Oh, one thing might matters.
If your multilingual training set includes both ZH and JA, I have no idea if there would be confusion of model facing same formed Hanzi or Kanji (as we just leverage characters and do not distinguish them from different languages)

**Comment by nuts-kun** [comment]
I took that into consideration when writing the preprocessing and dataset class, so I think it should be fine ;)

**Comment by lbdave94** [comment]
@nuts-kun can I ask you which is your batch size and the learning rate you used?
